### PR TITLE
fix cat index info to slice the right columns

### PR DESCRIPTION
**Pull request by vrecan**

### CODE REVIEW

File: lib/catindexinfo.go [hunk 1]
```diff
@@ -8,50 +8,53 @@ import (
...
 var ErrInvalidIndexLine = errors.New("Cannot parse indexline")
 
-// Create an IndexInfo from the string _cat/indices would produce
+//Create an IndexInfo from the string _cat/indices would produce
+//EX: health status index    pri rep docs.count docs.deleted store.size  pri.store.size
+//green open logs-2015-06-19   2 0   135389346  0            53048922233 53048922233
 func NewCatIndexInfo(indexLine string) (catIndex *CatIndexInfo, err error) {
 	split := strings.Fields(indexLine)
-	if len(split) < 4 {
+	if len(split) < 5 {
```
Comment (weberr13) [inline lib/catindexinfo.go:16]: this breaks backwards compatibility with 1.3, but I guess that is the price we pay.  The only other way would be to check the contents of split[1] and see if it matched the pattern "open" "closed" then we assume this behavior, if not we use the old behavior.

**Comment by weberr13** [comment]
:+1: backwards compatibility would make me happier.

File: lib/catindexinfo.go [hunk 2]
```diff
@@ -61,7 +64,8 @@ func NewCatIndexInfo(indexLine string) (catIndex *CatIndexInfo, err error) {
 // Pull all the index info from the connection
 func (c *Conn) GetCatIndexInfo(pattern string) (catIndices []CatIndexInfo) {
 	catIndices = make([]CatIndexInfo, 0)
-	args := map[string]interface{}{"bytes": "b"}
+	//force it to only show the fileds we know about
+	args := map[string]interface{}{"bytes": "b", "h": "health,status,index,pri,rep,docs.count,docs.deleted,store.size,pri.store.size"}
```
Comment (vrecan) [inline lib/catindexinfo.go:68]: this should fix the backwards compatibility issue and also make it harder to break in future versions where they add new fields.

**Comment by vrecan** [comment]
this should be ready for merge.

**Pull request merged**